spec/faked_project/lib/faked_project/some_class.rb

```diff
@@ -11,11 +11,9 @@ def reverse
   end
 
   def compare_with(item)
-    if item == label
-      return true
-    else
-      fail "Item does not match label"
-    end
+    return true if item == label
```

Comment:
Sure. Would you like me to disable that cop completely or special-case this file?